### ISSUE TITLE
Not compatibale

### ISSUE THREAD

**Issue by Justop2024** [NebulaModTeam/nebula#699]
Mod is not compatible with lastest update or at least errors out when we try to play

**Comment by Justop2024** [comment]
```ini
An error has occurred! Game version 0.10.30.23272
27 Mods used: [DSPBigFormingSize0.1.0] [DSPBigMechaStorage1.0.6] [DSP Mod Save1.2.1] [DSPMarker0.0.9] [UnlimitedFoundations1.0.3] [FastAbsorption1.0.0] [Place the pump anywhere. Pumps whatever ocene type the planet has.1.0.0] [LSTM0.9.0] [PlanetFinder1.1.3] [IlLine1.0.0] [LDBTool3.0.1] [DSP Common API1.6.5.0] [NebulaMultiplayerMod0.9.7.2] [NebulaMultiplayerModApi2.0.0.24] [BulletTime1.4.8] [ReorderTechQueue1.2.0] [SphereEditorTools2.2.3] [PlanetMiner3.0.9] [Common API Nebula Compatibility1.6.5.0] [EternalSolarSails1.0.0] [MoreMegaStructure1.5.5] [NebulaCompatibilityAssist0.4.14] [ProliferatorMultiplier1.0] [Bulldozer1.1.8] [FactoryLocator1.3.0] [PlanetwideSpray1.1.1] [Power_Network_Manager0.0.7] 
[Nebula Error] RefreshAstroBox_Transpiler failed. Mod version not compatible with game version.
```

**Comment by Justop2024** [comment]
```ini
An error has occurred! Game version 0.10.30.23272 (Host)
27 Mods used: [DSPBigFormingSize0.1.0] [DSPBigMechaStorage1.0.6] [DSP Mod Save1.2.1] [DSPMarker0.0.9] [UnlimitedFoundations1.0.3] [FastAbsorption1.0.0] [Place the pump anywhere. Pumps whatever ocene type the planet has.1.0.0] [LSTM0.9.0] [PlanetFinder1.1.3] [IlLine1.0.0] [LDBTool3.0.1] [DSP Common API1.6.5.0] [NebulaMultiplayerMod0.9.7.2] [NebulaMultiplayerModApi2.0.0.24] [BulletTime1.4.8] [ReorderTechQueue1.2.0] [SphereEditorTools2.2.3] [PlanetMiner3.0.9] [Common API Nebula Compatibility1.6.5.0] [EternalSolarSails1.0.0] [MoreMegaStructure1.5.5] [NebulaCompatibilityAssist0.4.14] [ProliferatorMultiplier1.0] [Bulldozer1.1.8] [FactoryLocator1.3.0] [PlanetwideSpray1.1.1] [Power_Network_Manager0.0.7] 
[Nebula Error] System.MissingFieldException: Field 'CraftData.prototype' not found.
  at NebulaWorld.Player.DroneManager.EjectMechaDroneFromOtherPlayer (NebulaModel.Packets.Players.PlayerEjectMechaDronePacket packet) [0x00092] ;IL_0092 
  at NebulaNetwork.PacketProcessors.Players.PlayerEjectMechaDroneProcessor.ProcessPacket (NebulaModel.Packets.Players.PlayerEjectMechaDronePacket packet, NebulaModel.Networking.NebulaConnection conn) [0x0002b] ;IL_002B 
  at NebulaModel.Packets.PacketProcessor`1[T].ProcessPacket (T packet, NebulaAPI.Networking.INebulaConnection conn) [0x00000] ;IL_0000 
  at NebulaModel.Networking.Serialization.NetPacketProcessor+<>c__DisplayClass22_0`2[T,TUserData].<SubscribeReusable>b__0 (NebulaModel.Networking.Serialization.NetDataReader reader, System.Object userData) [0x00018] ;IL_0018 
  at NebulaModel.Networking.Serialization.NetPacketProcessor.ReadPacket (NebulaModel.Networking.Serialization.NetDataReader reader, System.Object userData) [0x00007] ;IL_0007 
  at NebulaModel.Networking.Serialization.NebulaNetPacketProcessor.ProcessPacketQueue () [0x00033] ;IL_0033 
```

**Comment by s1rlon** [comment]
https://github.com/NebulaModTeam/nebula/pull/701 has been merged to master so I guess it's a matter of making a new release.

**Comment by starfi5h** [comment]
Fix in v0.9.8
